src/core/error.ml

```diff
@@ -146,6 +151,8 @@ module BetterErrors = struct
 				add_access FunctionReturn;
 			| Invariant_parameter i ->
 				add_access (TypeParameter i);
+			| Unify_custom _ ->
+				add_extra err
```

Comment:
What is this actually doing? It seems to mirror the handling of `acc_messages` exactly, so I don't see what difference this additional field makes.

Reply:
`acc_messages` are used to determine the "main error" and stuff like that. Keeping it separate avoids some strange mixup but I agree I don't like it much...

Reply:
Ah right, that actually makes a difference then. But shouldn't it appear in the haxe/want version as well in that case?

Reply:
I tried to add it to `Some access_next` below  but uh it got ignored for some reason